internal/v1/handler.go

```diff
@@ -258,6 +259,40 @@ func (h *Handlers) GetComposeStatus(ctx echo.Context, composeId uuid.UUID) error
 	if err != nil {
 		return err
 	}
+	// Remove password from user customization
+	// NOTE: Maybe not neccessary? The password was sent hashed in the first place
```

Comment:
This is not a good place to do it I think :)
We should mostly do it here IMO: https://github.com/osbuild/image-builder/blob/dad73f279a803bc4ec8fb3799b6a799e5b35eaf7/internal/v1/handler_compose_image.go#L67

Reply:
yeah thats what I did in https://github.com/osbuild/image-builder/blob/488b509c83a5da85a055122bd11b65daa077c4a9/internal/v1/handler_compose_image.go#L673-L709

but I will simplify it without the `oneOf` schema

Reply:
removed, since tests are still passing, password should be stored hashed if passed as plaintext and any subsequent operation like composing an image should only wokr with already hashed password.